audb/core/load.py

```diff
@@ -325,10 +325,16 @@ def job(path: str):
         src_path = audeer.path(db_root_tmp, path)
         dst_path = audeer.path(db_root, path)
         audeer.mkdir(os.path.dirname(dst_path))
-        audeer.move_file(
-            src_path,
-            dst_path,
-        )
+        if not os.path.isdir(src_path):
+            audeer.move_file(
+                src_path,
+                dst_path,
+            )
+        else:
+            # attachments can be folders
+            # which cannot be moved by audeer.move_file()
+            # under Windows
+            shutil.move(src_path, dst_path)
```

Comment:
As it turns out `audeer.move_file()` does fail under Windows when trying to move a folder.

Reply:
Why was it working before?

Reply:
I mean, in other places we are already relying on `audeer.move_file()` and never had a problem with the Windows tests.

Reply:
I checked and the desired destination folder did not exist before calling `audb.load()`.

Reply:
I think we should try to find out why as using `shutil.move()` could be dangerous. Since it takes longer than `os.replace()` it's probably not an atomic operation. Maybe it copies the files and afterwards deletes them from the source folder. This bears the risk that - in case the operation is interrupted - a folder is only partly copied. But since we only check for the existance of a folder to determine if a attachment was loaded, we would not recognize that.

Reply:
But of cause here we also have the problem that it is not an atomic operation

Reply:
Maybe we should start by adding additional tests in `audeer.move_file()` to find out what's going on?

Reply:
Yes, maybe we should try to fix it there as well.

Reply:
The problem was indeed that the folder existed already under `dst_path` as before we did:

```python
utils.mkdir_tree(paths, db_root)
```
which I now removed from the code.
Now `audeer.move_file()` does also work for a directory under Windows.

I also created https://github.com/audeering/audeer/issues/98to track this in `audeer`.

Reply:
BTW, I still don't know why this error did not happen before as we did used attachments with folders in the tests already.

Reply:
The difference was that before we stored attachments in the same subfolder, e.g.
```python
db.attachments['file'] = audformat.Attachment('extra/file.txt')
db.attachments['folder'] = audformat.Attachment('extra/folder')
```
But here I store them at separate places directly in the root of the database:
```python
db.attachments['file'] = audformat.Attachment('file.txt')
db.attachments['folder'] = audformat.Attachment('folder/')
```

Reply:
Ah ok, great we have a proper explanation :)

Reply:
What is the proper explanation? I don't see why the difference storage folders should make a difference?

Reply:
Mhh, right. Then we are back to 

![image](https://user-images.githubusercontent.com/10383417/233101733-4df70ea1-2ce7-4646-a32a-0e47f8b00846.png)

:)